src/turbolinks/history.coffee

```diff
@@ -20,16 +23,27 @@ class Turbolinks.History
 
   push: (location, restorationIdentifier) ->
     location = Turbolinks.Location.wrap(location)
+    locationStack.push(location)
     @update("push", location, restorationIdentifier)
 
   replace: (location, restorationIdentifier) ->
     location = Turbolinks.Location.wrap(location)
     @update("replace", location, restorationIdentifier)
 
+  back: (location, restorationIdentifier) ->
```

Comment:
Turbolinks actions are verbs. `back` is generally not used as a verb, so perhaps another word could be used for this action. `return`?